packages/common-lib/src/services/commentRegistryService.js

```diff
@@ -0,0 +1,88 @@
+import { get, post, update as coreUpdate } from './RestClient'
```

Comment:
is comments API generic to any entity or it is only for wrksheet?
if it is only for worksheet then why it can't be part of worksheet service?
API endpoint should also be like worksheet/<worksheetid>/comments